src/sql/workbench/common/constants.ts

```diff
@@ -51,7 +51,7 @@ export const NBFORMAT = 4;
 export const NBFORMAT_MINOR = 2;
 
 export const enum NotebookLanguage {
-	Notebook = 'Notebook',
+	Notebook = 'notebook',
```

Comment:
the language editor lookup code now uses the language ID instead of the language name.